cmd/cnbBuild_test.go

```diff
@@ -23,18 +23,22 @@ func newCnbBuildTestsUtils() cnbutils.MockUtils {
 }
 
 func addBuilderFiles(utils *cnbutils.MockUtils) {
```

Comment:
I don't know if we need to still call this `addBuilderFiles ` because we are not adding multiple once